src/backend/app/tasks/task_routes.py

```diff
@@ -86,3 +89,43 @@ async def new_event(
         user_data,
         background_tasks,
     )
+
+
+@router.post("/upload/{project_id}/{task_id}")
+async def upload_orthophoto_to_oam(
+    project_id: uuid.UUID,
+    task_id: uuid.UUID,
+    user_data: Annotated[AuthUser, Depends(login_required)],
+    project: Annotated[
+        project_schemas.DbProject, Depends(project_deps.get_project_by_id)
+    ],
+):
+    """
+    Uploads an orthophoto (TIFF) file to OpenAerialMap (OAM).
+
+    Args:
+        project_id: The UUID of the project.
```

Comment:
You only really need to add docstrings with `Args` and `Returns` on crud functions. For routes, this is automatically documented using type hints and OpenAPI spec 👌 

Plus docstrings are useful for **devs** to consume functions, but the route is the top level of the logic and nobody is importing and using the routes elsewhere